guidelines/sc/22/focus-not-obscured-minimum.html

```diff
@@ -7,5 +7,7 @@ <h4>Focus Not Obscured (Minimum)</h4>
     
     <p>When a <a>user interface component</a> receives keyboard focus, the component is not entirely hidden due to author-created content.</p>
 
-    <p class="note">If the interface is configurable so that the user can reposition content such as toolbars and non-modal dialogs, then only the initial positions of user-movable content would be considered for testing and conformance of this Success Criterion.</p>
+    <p class="note">Where content in a configurable interface can be repositioned by the user, then only the initial positions of user-movable content is considered for testing and conformance of this Success Criterion.</p>
+
+    <p class="note">Where content disclosed by the <em>user</em> can obscure other content, if the user can reveal the item with focus without moving keyboard focus (for example, by dismissing with Esc), such disclosed content is not considered to fully obscure the item receiving focus.</p>
```

Comment:
@mbgower thanks for making this two notes.  I do not think "disclosed" is a great word to use this paragraph.  My suggestion:

Where user interaction can cause new content to obscure other content, if the user can reveal the item with focus without moving keyboard focus (for example, by dismissing with <kbd>Esc</kbd>), that new content is not considered to be fully obscuring the item receiving focus.

Reply:
Thanks, @bruce-usab I'm concerned that the suggested text "Where user interaction can cause new content to obscure other content..." may be a little broad. We want to 'except' content the user directly opens. We don't want this to be interpreted as meaning that anything that occurs as a result of any user interaction (e.g., tabbing through the UI, populating a user input, answering a prompt, etc) could lead to a loss of focus. Obviously if the user isn't aware an 'interaction' of theirs has potentially obscured focus, they won't know to 'uncover' it.
Other suggestions welcome!

Reply:
I agree that the term "disclose" is odd in an interface context. How about "opens", or "reveals"?

Reply:
@mbgower I concede that my phrasing could be too broad.  I was trying to introduce as few new words as possible, and make the exception a little more active voice.  "Reveals" seems better (more generic) to me than "opens"..